src/algorithms/policy_gradient/ppo.jl

```diff
@@ -256,8 +261,12 @@ function _update!(p::PPOPolicy, t::AbstractTrajectory)
                 v′ = AC.critic(s) |> vec
                 if AC.actor isa GaussianNetwork
                     μ, σ = AC.actor(s)
-                    log_p′ₐ = normlogpdf(μ, σ, a)
-                    entropy_loss = mean((log(2.0f0π) + 1) / 2 .+ log.(σ))
+                    if ndims(a) == 2
+                        log_p′ₐ = sum(normlogpdf(μ, σ, a), dims=1)
```

Comment:
Note that logpdf of the univariate normal distribution is different from the multi-variate normal distribution.

Currently, normlogpdf only applies to the univariate normal distribution

https://github.com/JuliaReinforcementLearning/ReinforcementLearningCore.jl/blob/664449240c0a7c6ce62b462c7ff2e2dd894fa949/src/extensions/Distributions.jl#L12

https://github.com/JuliaStats/Distributions.jl/blob/7120d503c269503b0037a047399452b596ab43d4/src/multivariate/mvnormal.jl#L127

Reply:
Yes, was thinking about how to solve that. 

The current approach is that since we only get per variable mean and variation (i.e. no covariance) we can simplify the mv normal logpdf a lot, and if I'm not entirely mistaken (did this by myself late in the evening) it turned out I can just use logpdf on the entire thing and sum over the result in the action direction. 

I have not had time to look at it since then, but will make sure it seems correct next time I sit with it. I also am not sure if it is reasonable to assume they are not correlated, but I'm not sure how one would add that in a sensible way in the gaussian model.

Reply:
👍 
Let me know when you think it's ready to merge.

Reply:
Will do!

And assuming no covariance this should be correct, and there is a much easier argument. If there is no covariance they are independent and the combined probability of all dimensions in the action is just the product of the individual. The logpdf of the total probability is then the logpdf of the product, i.e. the sum of the individual logpdfs. Then again, this is assuming no covariance which is what we get from the gaussian model, but I'm not sure if there are better ways of doing it.

Reply:
> Then again, this is assuming no covariance which is what we get from the gaussian model, but I'm not sure if there are better ways of doing it.

I think this implementation is still very useful. Could you add this note in the docstring of `PPOPolicy`?

Reply:
Added something, didn't know exactly what you wanted added (i.e. how in detail it should be)? Feel free to update it as you want.